src/main/sources/VcfDataSource.js

```diff
@@ -91,13 +92,27 @@ function createFromVcfFile(remoteSource: VcfFile): VcfDataSource {
   return o;
 }
 
-function create(data: {url:string}): VcfDataSource {
+function create(data: Object): VcfDataSource {
   var url = data.url;
-  if (!url) {
-    throw new Error(`Missing URL from track: ${JSON.stringify(data)}`);
+  var content = data.content;
+  if (url!==null && url!== undefined) {
+    return createFromVcfFile(new VcfFile(new RemoteFile(url)));
   }
+  if (content!==null && content!== undefined) {
+    return createFromVcfFile(new VcfFile(new LocalStringFile(content)));
+  }
+  throw new Error(`Missing URL from track: ${JSON.stringify(data)}`);
+}
 
-  return createFromVcfFile(new VcfFile(new RemoteFile(url)));
+function create(data: {url?: string, content?: string}): VcfDataSource {
```

Comment:
thanks for adding this implementation in. Can you remove the old one from the source code?